katas/.editorconfig

```diff
@@ -0,0 +1,276 @@
+# EditorConfig is awesome:http://EditorConfig.org
```

Comment:
I have compared running `dotnet format` with and without these rules, and looks like this `.editorconfig` might be missing some rules as it fixes less formatting issues
![image](https://user-images.githubusercontent.com/4606607/200999735-415a2623-a9a1-4386-a01e-fe1a5fed9ce2.png)

If possible, please verify and compare if we are missing any rules here. We can also remove this file for now and add it later once we figure out which rules we want to add/remove in the way

Reply:
We tested the new file and confirmed more changes with the .editorconfig rules
In conclusion, its better to keep the file with the updated changes